workers/tasks/export/html.tsx

```diff
@@ -166,12 +166,14 @@ const renderFrontMatterForPandoc = (
 		primaryCollectionTitle,
 		doi,
 		licenseSlug,
+		publisher,
 	},
 	targetPandoc,
 ) => {
 	const pandocFormatsWithoutTemplate = ['docx', 'plain', 'odt'];
+	// do not put community title if this is a book
 	const communityAndCollectionString =
-		communityTitle + (primaryCollectionTitle ? bullet + primaryCollectionTitle : '');
+		(!publisher ? communityTitle + bullet : '') + (primaryCollectionTitle || '');
```

Comment:
Super small thing, but it's a good practice to try and keep conditionals/ternaries positive, e.g. instead of saying "if there is no publisher, do Y, otherwise X", the code should read "if there is a publisher, do X, otherwise Y". Don't let this block your PR from moving forward though!

Reply:
i just changed it, trying to be different for no reason 😅